include/model.h

```diff
@@ -142,6 +142,14 @@ typedef enum ShapeTypes {
     SHAPE_TYPE_SPECIAL_GROUP = 10,
 } ShapeTypes;
 
+typedef enum ExtraTileTypes {
+    EXTRA_TILE_0 = 0,
```

Comment:
These are known and should be
EXTRA_TILE_NONE = 0,
EXTRA_TILE_MIPMAPS = 1,
EXTRA_TILE_AUX_SAME = 2, // has aux tile with same settings as main
EXTRA_TILE_AUX_INDEPENDENT = 3, // has aux tile with different settings